src/base/noscript.scss

```diff
@@ -8,6 +8,12 @@
 	display: block !important;
 }
 
+@media screen {
+	%noscript-screen-display-block-important {
```

Comment:
@duboisp @LaurentGoderre I'll revise this PR in the following ways to address your concerns.
* Move my placeholder selector into the menu plugin's noscript file
* Prefix my placeholder selector's name with ``menu-``
* Nest the screen media query inside my placeholder selector